processes/consumer/process.go

```diff
@@ -60,7 +60,10 @@ func (p processArgs) process(ctx context.Context, cfg config.Config, inMemDB *mo
 	}
 
 	tags["op"] = _event.Operation()
-	evt := event.ToMemoryEvent(_event, pkMap, topicConfig.tc, cfg.Mode)
+	evt, err := event.ToMemoryEvent(_event, pkMap, topicConfig.tc, cfg.Mode)
+	if err != nil {
```

Comment:
@Tang8330 Do we want to set `tags["what"]` here?

Reply:
`to_mem_event_err`?